### PR TITLE
Pysnooper recipe

### PR DESCRIPTION
**Pull request by volpatto**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by volpatto** [comment]
The LICENSE is not being included in the MANIFEST.in in the PySnooper. Due to it, the CI tests are not passing. I will fix it with package dev team.

**Comment by volpatto** [comment]
For now, I will "hard include" the LICENSE in the recipe directory. If the update of the package succeed, then I update also the recipe properly.

File: recipes/pysnooper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+test:
+  imports:
+    - pysnooper
+    - pysnooper.third_party
+
+about:
+  home: https://github.com/cool-RR/PySnooper
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (chrisburr) [inline recipes/pysnooper/meta.yaml]: Conda automatically looks in the recipe directory if it isn’t found in the work directory:
```suggestion
  license_file: LICENSE
```

**Comment by volpatto** [comment]
@conda-forge/staged-recipes @conda-forge/python please anyone could review and eventually guide the merge for this PR? It's a pure Python package. Thanks!

**Comment by volpatto** [comment]
Oh, I closed it accidentally, sorry.

**Pull request merged**